src/features/applicationMetadata/types.ts

```diff
@@ -12,26 +12,20 @@ export interface IncomingApplicationMetadata {
   org: string;
   partyTypesAllowed: IPartyTypesAllowed;
   dataTypes: IDataType[];
-  altinnNugetVersion?: string;
   autoDeleteOnProcessEnd: boolean;
-  onEntry?: IOnEntry;
   features?: Partial<IBackendFeaturesState>;
   promptForParty?: 'always' | 'never';
+  externalApiIds?: string[];
```

Comment:
Expanding the `applicationmetadata` file is a bit tricky, since so many different packages use this file, and we've seen before that we quickly end up in a situation where one package (i.e. Studio) removes unknown properties and re-deploys the file without them.

But what we _did_ do is adding custom stuff just between app-frontend-react and app-lib-dotnet, such as the `altinnNugetVersion` property. So when the backend reads the file from disk, it will add in bits and bobs we need, including that version number, feature flags enabled for the app, etc. So maybe that's the best way forwards? I.e. get the backend to generate a list of allowed external apis from what's implemented, and return those together with the rest of the metadata. Saves us a request as well. :raised_hands:

Reply:
Yes, that would be much better than having the application developers define it 👌